packages/sodium/src/js/modules/AbemaTVLiveTypeHandler.js

```diff
@@ -44,8 +139,9 @@ export default class AbemaTVLiveTypeHandler extends GeneralTypeHandler {
 
   // eslint-disable-next-line class-methods-use-this
   get_video_thumbnail() {
-    /* あとから見ることができるような、固定されたサムネイルはなさそう */
-    return "";
+    return `https://hayabusa.io/abema/channels/logo/${
+      location.pathname.split("/").slice(-1)[0]
```

Comment:
```suggestion
      location.pathname.split("/").at(-1)
```

https://developer.mozilla.org/ja/docs/Web/JavaScript/Reference/Global_Objects/Array/at